packages/core/src/indexing/database-index-api.ts

```diff
@@ -81,6 +88,25 @@ export abstract class DatabaseIndexApi<DateType = Date | number> {
     this.syncApi = api
   }
 
+  /**
+   * Add a composite index to the database
```

Comment:
Agreed with Paul on avoiding the term composite within js-ceramic as js-ceramic knows nothing of composites.